django/urls/__init__.py

```diff
@@ -1,12 +1,13 @@
 from .base import (
     clear_script_prefix, clear_url_caches, get_script_prefix, get_urlconf,
-    is_valid_path, resolve, reverse, reverse_lazy, set_script_prefix,
-    set_urlconf, translate_url,
+    is_valid_path, path, re_path, resolve, reverse, reverse_lazy,
+    set_script_prefix, set_urlconf, translate_url,
 )
+from .converters import register_converter
 from .exceptions import NoReverseMatch, Resolver404
 from .resolvers import (
-    LocaleRegexProvider, LocaleRegexURLResolver, RegexURLPattern,
```

Comment:
Looking at the usage, it's to specify the *type* of the return value of the `url` function.

In my opinion: depending on the fact that `url` returns an object that is an instance of `LocaleRegexProvider` is deep into the territory of over-specification.

Other usage:

* https://github.com/arteria/django-compat/blob/master/compat/__init__.py: A django-compat layer. Does not really signify a usage, I think.
* https://github.com/django-polymorphic/django-polymorphic/blob/master/polymorphic/admin/parentadmin.py: Only in a comment.